src/components/multi-accordion-with-title/MultiAccordionWIthTitle.tsx

```diff
@@ -39,7 +42,10 @@ const MultiAccordionWithTitle: FC<MultiAccordionWithTitleProps> = ({
   }
 
   const accordionTitle = title && (
-    <Typography sx={sx.title} variant={VariantEnum.H5}>
+    <Typography
+      sx={sx.title}
+      variant={isMobile ? VariantEnum.H6 : VariantEnum.H5}

```

Comment:
also, you can do this in styles and do not export this enum

Reply:
yeap, thx